app/layout.tsx

```diff
@@ -1,22 +1,33 @@
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
-import "./globals.css";
+import "./global.css";
+import { Button, Paper, buttonGroupClasses } from "@mui/material";
+import Image from "next/image";
 
-const inter = Inter({ subsets: ["latin"] });
-
-export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+export const metadata = {
+	title: "Game Trucker",
+	description: "Generated by Next.js",
 };
 
-export default function RootLayout({
-  children,
-}: Readonly<{
-  children: React.ReactNode;
-}>) {
-  return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
-    </html>
-  );
+export default function RootLayout({ children }: { children: React.ReactNode }) {
+	return (
+		<>
+			<html lang="en">
+				<body>
+					<div style={{ width: "100vw", height: "100vh", position: "relative" }}>
```

Comment:
You already have mui installed, you can utilise `Box` component instead of native HTML `div`, that would help a bit with consistency how you're applying your styles to elements across whole app

Reply:
Consider using `Stack` or `Grid` components for more complex layouts, it's easier to make responsive UI this way